internal/database/sqlc/models.go

```diff
@@ -3,11 +3,30 @@
 package sqlc
 
 import (
+	"database/sql"
 	"time"
+
+	"github.com/google/uuid"
 )
 
 type Commandment struct {
 	Number      int32     `json:"number"`
 	Phrase      string    `json:"phrase"`
 	CreatedDate time.Time `json:"createdDate"`
 }
+
+type User struct {
+	ID                     uuid.UUID      `json:"id"`
+	SubjectID              string         `json:"subjectID"`
```

Comment:
Question: What's the difference between string and sql.NullString?

Reply:
`sql.NullString` is the way go has chosen to represent a nullable database string. `*string` could've worked just as well, but they didn't go that route for some reason.

Reply:
yes, but it was still a legit question.